app/src/main/java/pl/szczodrzynski/edziennik/ui/modules/timetable/TimetableDayFragment.kt

```diff
@@ -276,6 +295,28 @@ class TimetableDayFragment : LazyFragment(), CoroutineScope {
             lb.detailsFirst.text = listOfNotEmpty(timeRange, classroomInfo).concat(bullet)
             lb.detailsSecond.text = listOfNotEmpty(teacherInfo, teamInfo).concat(bullet)
 
+            val typeIcon = when (attendance?.baseType) {
+                Attendance.TYPE_PRESENT, Attendance.TYPE_PRESENT_CUSTOM -> CommunityMaterial.Icon.cmd_check_circle_outline to R.color.md_green_500
+                Attendance.TYPE_ABSENT -> CommunityMaterial.Icon.cmd_close_circle_outline to R.color.md_red_500
+                Attendance.TYPE_ABSENT_EXCUSED -> CommunityMaterial.Icon.cmd_circle_edit_outline to R.color.md_blue_500
+                Attendance.TYPE_RELEASED -> CommunityMaterial.Icon.cmd_account_supervisor_circle_outline to R.color.md_lime_500
+                Attendance.TYPE_BELATED -> CommunityMaterial.Icon.cmd_clock_alert_outline to R.color.md_orange_600
+                Attendance.TYPE_BELATED_EXCUSED -> CommunityMaterial.Icon.cmd_clock_check_outline to R.color.md_yellow_500
+                Attendance.TYPE_DAY_FREE -> CommunityMaterial.Icon3.cmd_sticker_circle_outline to R.color.md_pink_400
+                else -> null
+            }
+
+            if (typeIcon != null) {
+                val (icon, color) = typeIcon
+                lb.attendanceIcon.setImageDrawable(
+                    IconicsDrawable(activity, icon).apply {
+                        colorRes = color
+                        sizeDp = 24
+                    }
+                )
+                lb.attendanceIcon.isVisible = true
+            }
+
```

Comment:
Change the icons, as voted by the community. Use colors from `AttendanceManager` - it has ready-to-use methods just for this purpose.